src/e2e/java/uk/gov/hmcts/reform/sscs/functional/AbstractFunctionalTest.java

```diff
@@ -121,15 +119,28 @@ protected SscsCaseData createCaseData() {
         return buildSscsCaseData(caseReference, "Yes", "Yes", SYA_APPEAL_CREATED, "oral");
     }
 
-    protected static String generateRandomCaseReference() {
+    protected String generateRandomCaseReference() {
         String epoch = String.valueOf(Instant.now().toEpochMilli());
-        return "SC" + epoch.substring(3, 6)
+        Map<String,String> sscCodeMap = regionalProcessingCenterService.getSccodeRegionalProcessingCentermap();
+        String scNumber = generateScNumber(Instant.now(),sscCodeMap);
```

Comment:
After we saw code in the caseloader repo assuming an option variable was always present, is there any reason the `sscCodeMap` would be null or unusable?